examples/searchcommands_app/package/bin/generatetext.py

```diff
@@ -35,6 +35,7 @@ class GenerateTextCommand(GeneratingCommand):
 
     def generate(self):
         text = self.text
+        self.logger.debug("Genearting text: %s" % self.text)
```

Comment:
typo: generating

Should we say `Generating %d events with text %s`?

Reply:
done